community/louvain.tex

```diff
@@ -25,16 +29,16 @@
 \end{equation}
 and 
 \begin{equation}
-	\delta(c_i,c_j) = \left \{ \begin{array}{ll}
-	1 & \text{if} \quad  c_i = c_j \\
+	\delta_{ij}(c) = \left \{ \begin{array}{ll}
+	1 & \text{if} \quad  c(i) = c(j) \\
 	0 & \text{otherwise} \end{array} \right ..
 \end{equation}
 
```

Comment:
Explanation of the `c(i)` notation can be put here. The word `current` is not necessary anymore, since it is already implicitly contained in `c`.